strings/tcg/set_one.json

```diff
@@ -83,10 +83,10 @@
 			"desc": "Prior to the introduction of on-station psychologists, the Bartender served to alleviate many employees' woes and fears. Remember, always drink responsibly.",
 			"rules": "",
 			"icon_state": "bartender",
-			"power": 0,
-			"resolve": 0,
-			"faction": "",
-			"summoncost": 1,
+			"power": "3",
```

Comment:
Why did these change from numbers to strings of numbers?

Reply:
These changed for the sake of equipment cards, which use the variable to list the change in stats they give (I.E. +1)

Reply:
fair enough